{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "c2f48fc8_1d096446",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1579231
      },
      "writtenOn": "2022-09-07T07:21:33Z",
      "side": 1,
      "message": "Disabling flaky (read()) test temporarily.",
      "revId": "e48559df75dc9711f9f86c5d637f4c0fde419f4c",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "fe18ad22_b7c1d611",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1003224
      },
      "writtenOn": "2022-09-07T15:33:06Z",
      "side": 1,
      "message": "so this is _another_ failure? we had the bind() one, the close one jmgao just sent a fix for, and there\u0027s still [at least] one more?\n\n(note that it\u0027s usual to just say _what_ tests you ran, not include the full output. but in the case of a test that _fails_ like this it might be useful to include the test output ... but the _failing_ output rather than the \"it didn\u0027t run because i deleted it\" output!)",
      "revId": "e48559df75dc9711f9f86c5d637f4c0fde419f4c",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}